components/Report.vue

```diff
@@ -190,27 +190,30 @@
         </div>
         <div v-if="!dataPresent" class="pb-3" />
       </section>
-      <section class="section" v-if="climateData">
+      <section class="section large-screen" v-if="climateData">
         <div id="temperature">
           <TempReport />
         </div>
       </section>
-      <section class="section" v-if="climateData">
+      <section class="section large-screen" v-if="climateData">
         <div id="precipitation">
           <PrecipReport />
         </div>
       </section>
-      <section class="section">
+      <section class="section large-screen">
         <div id="permafrost" v-if="permafrostData || type != 'latLng'">
           <PermafrostReport />
         </div>
       </section>
-      <section class="section" v-if="flammabilityData || vegChangeData">
+      <section class="section large-screen" v-if="flammabilityData || vegChangeData">
         <div id="wildfire">
           <WildfireReport />
         </div>
         <BackToTopButton />
       </section>
+      <section class="section !large-screen">
+          <span class="centered"><b-icon icon="phone-rotate-landscape" size="is-medium" />&nbsp;&nbsp;To see additional charts and data, please rotate your device into landscape mode.</span>
```

Comment:
Add `is-size-5` to the CSS here to make the font size match other sections when the charts are hidden.